README.md

```diff
@@ -48,3 +48,113 @@ A Spring Boot server for collecting, storing, and analyzing error logs with REST
 * **Extensibility:** Pluggable architecture for custom AI models or storage backends.
 * **Reliability:** Fault-tolerant with retries and backup mechanisms.
 
+## Setup Guide
+
+### Prerequisites
+
+- Java 17 or higher
+- Maven 3.8 or higher
+- PostgreSQL 13 or higher (or your preferred database)
+- Docker (optional, for containerized deployment)
```

Comment:
Kindly change targets to:
- Java 21 instead of 17
- SQLite in place of PostgresSQL